pom.xml

```diff
@@ -51,9 +51,9 @@
     </issueManagement>
 
     <properties>
-        <maven.compiler.target>11</maven.compiler.target>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.release>17</maven.compiler.release>
```

Comment:
We definitely don't want this :)

Reply:
I changed it to get my Test using "record" running. Is there another way to enable Java version with "record"??

Reply:
I would say let's remove the test altogether as it's not a Quarkus issue, it's a Jackson issue so we should not be testing for it (unless we have some problematic integration point)

Reply:
Is there a plan to update the maven.compiler settings to a newer version than 11? 
So far i will remove the Test.

Reply:
> Is there a plan to update the maven.compiler settings to a newer version than 11?

No.

But for Java 17 specific things, we do have a dedicated integration test.

Reply:
If it’s a Jackson bug, I don’t think we need a test in our code base.

Reply:
Okay, than next time we need to check again manually...

Reply:
If we were to add such tests, we would need to add them for every feature under the sun, that just isn't practical. 
What we do test extensively is our integration points with the various libraries and our workarounds for known problems. 

The Jackson problem you linked to is neither - but of course we do want to bump to the latest Jackson version.